### ISSUE TITLE
Path failures on Windows

### ISSUE THREAD

**Issue by nshahan** [dart-lang/sdk#37490]
I believe these failures have the same root cause of incorrect paths on windows platforms. Feel free to break out into separate issues if they are not.

DDK compile failing on Devtools windows Travis bot:
https://travis-ci.org/flutter/devtools/jobs/556885465

DDK test bot failures:
https://ci.chromium.org/p/dart/builders/ci.sandbox/ddc-win-release-chrome/5184

**Comment by Markzipan** [comment]
Fix out at https://dart-review.googlesource.com/c/sdk/+/108503

**Comment by nshahan** [comment]
@Markzipan It looks like the fix landed. Are you expecting any other changes here? 

@jacob314 Everything good now on the devtools bots?

**Comment by Markzipan** [comment]
No more windows-specific changes planned for now (unless something goes awry).

**Comment by nshahan** [comment]
The flutter devtools bot is still showing a failure.  https://travis-ci.org/flutter/devtools/jobs/562805532 but the change from @Markzipan was not part of the 2.5.0-dev.1.0 release. 

Leaving this issue open until the next release and we can confirm the fix in the devtools repo.

**Comment by vsmenon** [comment]
Is there anything to do here?

**Comment by nshahan** [comment]
This is an issue that flutter web users are hitting and we think it has been resolved but it is unclear when the fix will actually be rolled into the flutter releases and visible to the end users to confirm.

**Comment by vsmenon** [comment]
@jacob314 - can we close this issue?  I think it triggered by a dev tools test failure?

**Comment by warenix** [comment]
Confirm fixed in 2.5.0-dev.2.1 release

```
λ dart --version
Dart VM version: 2.5.0-dev.2.1 (Thu Aug 15 16:05:39 2019 +0200) on "windows_x64"
```
